Dockerfile.debug

```diff
@@ -0,0 +1,43 @@
+ARG RELEASE_MODE=community
+ARG TARGETARCH=amd64
+
+FROM golang:1.16 as builder
```

Comment:
maybe we should consider to add the below line before FROM:
ARG GO_VERSION=1.16 
and use it in FROM